frontend/src/components/common/videos-block/videos-block.tsx

```diff
@@ -49,6 +49,10 @@ const VideosBlock: FC<VideoBlockProps> = ({ blockTitle, videoCards, isLazyBlock,
       return numberOfVideoForUpload;
     }
 
+    if (numberOfUploadedVideos > numberOfVideoForUpload) {
```

Comment:
this code not ideal, because it will break other logic, please find a universal solution for this problem

Reply:
done